datasets/add_rounding_instructions.py

```diff
@@ -0,0 +1,27 @@
+import json
```

Comment:
Would it be easier and more straightforward to define it as a function inside datasets/utils.py and then import it, so that you don't need to call submodule commands and also don't need to read from the disk and can modify data in-memory?